protocol/test/amo/oud/oud-token.ts

```diff
@@ -0,0 +1,210 @@
+import { expect } from "chai";
+import { OudToken, OudToken__factory } from "../../../typechain";
+// import { recoverToken, shouldRevertNotOwner, testErc20Permit } from "../../helpers";
+import { loadFixture } from "@nomicfoundation/hardhat-network-helpers";
+import { BigNumberish, Signer, TypedDataDomain, TypedDataField } from "ethers";
+import { ethers } from "hardhat";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers"
+import { splitSignature } from "ethers/lib/utils";
+import { blockTimestamp } from "../../helpers";
+ 
+describe("Mintable Token", async () => {
+    let token: OudToken;
+    let owner: SignerWithAddress;
+    let minter: SignerWithAddress;
+    let alan: SignerWithAddress;
+    let spender: SignerWithAddress;
```

Comment:
Unused - can remove?

Reply:
want me to refactor?
used various places such as https://github.com/TempleDAO/temple/blob/1b2bc9333ca7600e4b7ade77661b0ccf58be2c47/protocol/test/amo/oud/oud-token.ts#L68

Reply:
no the `spender` variable is unused AFAIK

Reply:
used now that permit tests are actually working